packages/rich-text/src/component/index.native.js

```diff
@@ -700,6 +701,8 @@ export class RichText extends Component {
 			this.firedAfterTextChanged = false;
 		}
 
+		const width = maxWidth && this.state.width && ( maxWidth - this.state.width ) < 10 ? maxWidth : this.state.width;
```

Comment:
Could you please add a comment what is happening here?

Reply:
+1

Reply:
Done